modules/nw-ovn-kubernetes-examine-nb-database-contents-ref.adoc

```diff
@@ -8,13 +8,19 @@
 
 The following table describes the command line arguments that can be used with `ovn-nbctl` to examine the contents of the northbound database.
 
+
+[NOTE]
+====
+Open a remote shell in the pod you wish to view the contents of and then run the `ovn-nbctl` commands.
```

Comment:
in which pod?

Reply:
I think pod should be node here? Does that make more sense?

Reply:
I will fire up cluster tomorrow and double check this. I did create short video on this too so I will look at that again.

Reply:
Pretty sure one of the pods but will verify tomorrow.

Reply:
This is one of the pods as retrieved from the command in the previous section `oc get po -n openshift-ovn-kubernetes` but I will verify tomorrow, but pretty sure on this. This note may simply reiterate what has come before with running ```
oc exec -n openshift-ovn-kubernetes -it ovnkube-master-gt4ms \
-c northd -- ovn-nbctl lb-list
``` so maybe some duplication there and it may not be needed at all, but oc exec is effectively similiar to opening a remote shell in the pod
--

Reply:
Ran some tests this command should be run against the identified raft leader pod as shown in the previous section. For example `oc exec -n openshift-ovn-kubernetes -it ovnkube-master-ts2f2 -c northd -- ovn-nbctl show`. To be honest, with all this detailed in the previous section, that note is probably not needed as this is all described in the previous section.